app/controllers/api/v1/users_controller.rb

```diff
@@ -2,4 +2,20 @@ class Api::V1::UsersController < ApplicationController
   def show
     render json: UserSerializer.new(User.find(params[:id]))
   end
+
+  def create
+    user = User.create(user_params)
+    if user.save
+      render json: UserSerializer.new(user), status: :created
+    else
+      render json: {error: user.errors.full_messages.to_sentence}, status: :unprocessable_entity
```

Comment:
I like this error message; I like that it'll give specific information about what field wasn't filled in.